### ISSUE TITLE
Check potential md to adoc conversion issue with html anchor

### ISSUE THREAD

**Issue by tbouffard** [bonitasoft/bonitasoft.github.io#64]
Seen with Bonita 7.5 conversion.

A lot of `a` anchor are not migrated

![image](https://user-images.githubusercontent.com/27200110/103771044-50887980-5027-11eb-9e77-fc0c4f57093e.png)

**Comment by alachambre** [comment]
More details: 

The following markdown

```
<a id="user-task"/>
### User task list re-design
```

Is converted to 

```
+++<a id="user-task">++++++</a>+++
=== User task list re-design
```

Which gives the following HTML: 

``` html
<div class="sect2">
	...
	<div class="paragraph">
		<p><a id="user-task"></a></p>
	</div>
</div>
<div class="sect2">
	<h3 id="_user_task_list_re_design">
		<a class="anchor" href="#_user_task_list_re_design"></a>
		User task list re-design
	</h3>
	<div class="paragraph">
	...
</div>
```

-> A native anchor is generated by Antora for Asciidoctor titles. 
-> Any links in the Bonita documentation that refers to the old anchor must be updated to the new one
⚠️ It seems to mostly impact the highlight cards in the index.html which refer a lot to anchors in the release note!

-> The empty links generated from the old anchors should be removed, but the impact seems to be small.

**Comment by benjaminParisel** [comment]
I think we could replace `+++<a id="user-task">++++++</a>+++` by `[#user-task]` to keep this anchor. I already do it for release note in branch 7.6 and 7.10 and it's work fine.

Automatically script cool be nice to be written and add in migration procedure

**Comment by tbouffard** [comment]
closed by #90
